### PR TITLE
Allow dict-style logging in breadcrumbs processing

### PR DESCRIPTION
**Pull request by rakai93**
This commit changes the breadcumbs processor to be able to handle dict-style logging properly, e.g. `logger.info('Hello %(who)s', {'who': 'world'})`. Prior to this change, breadcrumbs shown in Sentry were not interpolated and thus hard to follow.

### CODE REVIEW

**Comment by rakai93** [comment]
Seems like the travis CI jobs is broken (correct me if I'm wong), but tests are failing unrelated to this change. Can someone please have a look?

**Comment by ashwoods** [comment]
@rakai93 mind rebasing from master?

**Comment by rakai93** [comment]
@ashwoods Thank you, I merged master and now the checks pass!
